__tests__/app.test.js

```diff
@@ -213,131 +213,289 @@ describe('PATCH commands', () => {
+describe('Get /api/reviews Queries', () => {
+    test('200: should return the reviews with the category dexterity', () => {
+        return request(app).get("/api/reviews?category=dexterity")
+        .expect(200)
+        .then(({body})=>{
+            const reviews = body.reviews;
+          
+            expect(reviews).toHaveLength(1);
+            expect(reviews[0]).toEqual({
+                    review_id : 2,
+                    title: 'Jenga',
+                    designer: 'Leslie Scott',
+                    owner: 'philippaclaire9',
+                    review_img_url:
+                      'https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700',
+                    review_body: 'Fiddly fun for all the family',
+                    category: 'dexterity',
+                    created_at: "2021-01-18T10:01:41.251Z",
+                    votes: 5,
+                    comment_count :"3"
+                  })                
+            })
         });
+
+    test('200: should return reviews with category euro game', () => {
+        return request(app).get("/api/reviews?category=euro+game")
+        .expect(200)
+        .then(({body})=>{
+            const reviews = body.reviews;
+          
+            expect(reviews).toHaveLength(1);
+            expect(reviews[0]).toEqual({
+                    review_id : 1,
+                    title: 'Agricola',
+                    designer: 'Uwe Rosenberg',
+                    owner: 'mallionaire',
+                    review_img_url:
+                        'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+                    review_body: 'Farmyard fun!',
+                    category: 'euro game',
+                    created_at: "2021-01-18T10:00:20.514Z",
+                    votes: 1,
+                    comment_count :"0"
+                  })                
+            })
     });
 
-    test('404: GET followed by invalid review datatype should return a message', () => {
-        return request(app)
-        .get("/api/reviews/bananas")
-        .expect(400)
+    test('200: should return reviews with category social deduction', () => {
+        return request(app).get("/api/reviews?category=social+deduction")
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("Bad Request - expected a number and got text e.g. received three instead of 3")
+            const reviews = body.reviews
+            expect(reviews).toHaveLength(11);
+            reviews.forEach((review)=>{
+                expect(review.category).toBe("social deduction")
+            })
         })
     });
 
-    test('400: GET follow by invalid review id ie resource that doesn\'t exist should return a message', () => {
-        return request(app)
-        .get("/api/reviews/99999")
-        .expect(404)
+    test('200: should return no reviews with category children\'s games', () => {
```

Comment:
This one can stay, but please name it more generically - we're not testing specifically for Children's Games (even though we are using it as an example in our tests) - we're testing more generally for an empty array when a category has no reviews.

Reply:
Still needs the description updating, please.